crawler/dataParser.py

```diff
@@ -37,6 +45,8 @@ def parserWorker(self):
 
       for link in toParse:
         obj = extractData(link)
+        self.findNewLinks(link)
```

Comment:
Tbh this doesn't need to get done, youre already getting all the "a" tags from extract data.